src/sentry/integrations/discord/__init__.py

```diff
@@ -11,4 +10,4 @@
 from .utils import *  # noqa: F401,F403
 from .views import *  # noqa: F401,F403
 
-rules.add(DiscordNotifyServiceAction)
+DiscordMessagingSpec().initialize()
```

Comment:
I would like that, especially in coordination with the work being done on the SCM and ticketing verticals.

I'm 👍 to nuking the `__init__` imports. Moreover, I'd like to move the `initialize()` call from here (initializing as a side effect of package import == 🤮) to somewhere like `sentry/runner/initializer.py` where it can be called at a predictable time during server start-up.